README.md

```diff
@@ -32,37 +32,30 @@ note: Run with `RUST_BACKTRACE=1` for a backtrace.
 ```txt
 Well, this is embarrasing.
 
-human-panic had a problem and crashed. To help us diagnose the problem you can
-send us a crash report.
+human-panic had a problem and crashed. To help us diagnose the problem you can send us a crash report.
 
-We have generated a report file at "<reports not generated yet>". Submit an
-issue or email with the subject of "human-panic Crash Report" and include the
-report as an attachment.
+We have generated a report file at "/var/folders/zw/bpfvmq390lv2c6gn_6byyv0w0000gn/T//report-8351cad6-d2b5-4fe8-accd-1fcbf4538792.toml". Submit an issue or email with the subject of "human-panic Crash Report" and include the report as an attachment.
```

Comment:
That double slash was fixed by you in #9  :)

Reply:
Ha, done!